runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8280,6 +8285,31 @@ done:;
 	}
 
 #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	void
+	defaultValueWithUnflattenedFlattenables(J9Class *clazz, j9object_t instance)
+	{
+		J9FlattenedClassCacheEntry * entry;
+		UDATA length = clazz->flattenedClassCache->numberOfEntries;
+		for (UDATA index = 0; index < length; index++) {
+			entry = J9_VM_FCC_ENTRY_FROM_CLASS(clazz, index);
+			if (!J9_ARE_ALL_BITS_SET(J9ClassIsFlattened, entry->clazz->classFlags)) {
+				if (entry->offset == UDATA_MAX) {
+					J9Class *definingClass = NULL;
+					J9ROMFieldShape *field = entry->field;
```

Comment:
set this one to NULL, create another field called `entryField` which is initialized to `entry->field`